rust/serde_bser/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "serde_bser"
+version = "0.1.0"
+authors = ["Siddharth Agarwal <sid0@fb.com>"]
+
+[dependencies]
+byteorder = "1.0"
+bytes = "0.4"
+error-chain = "*"
```

Comment:
Using `"*"` versions is generally bad form for Cargo.toml. crates.io won't let you upload anything with a `"*"` version constraint in it.